src/pedestal_toolbox/cors.clj

```diff
@@ -0,0 +1,7 @@
+(ns pedestal-toolbox.cors)
+
+(defn domain-matcher-fn [list-of-strings]
+  (let [patterns (map re-pattern list-of-strings)]
```

Comment:
Yeah, I think this function should expect regular expressions because what resource-conifg supports is outside the concern of this library.